docs/src/misc.rst

```diff
@@ -227,6 +227,11 @@ API
 
     Changes the current working directory.
 
+.. c:function:: int uv_homedir(char* home, size_t size)
+
+    Gets the current user's home directory. Calls `SHGetKnownFolderPath()` on
+    Windows, and `getlogin()` followed by `getpwnam()` otherwise.
```

Comment:
We can now document functions with automagic links to man pages:

":man:`getaddrinfo(3)`"